src/sysinfo/nvapi_adapter.cpp

```diff
@@ -206,6 +206,11 @@ namespace dxvk {
         return m_dxgiDesc.DedicatedVideoMemory / 1024;
     }
 
+    uint32_t NvapiAdapter::GetVirtualVRamSize() const {
+        // Report VRAM size from DXVK to honor memory overrides
+        return (m_dxgiDesc.DedicatedVideoMemory + m_dxgiDesc.DedicatedSystemMemory) / 1024;
```

Comment:
In reality `DedicatedSystemMemory` will always be zero, see https://github.com/doitsujin/dxvk/blob/master/src/dxgi/dxgi_adapter.cpp#L371